lib/create-contentful-api.ts

```diff
@@ -30,19 +29,19 @@ export interface ContentfulClientApi {
 
   getAsset(id: string): Promise<Asset>
 
-  getAssets(query?: FieldsQueries<AssetFields>): Promise<AssetCollection>
+  getAssets(query?: EntryQueries<AssetFields>): Promise<AssetCollection>
```

Comment:
Maybe we should alias `AssetQueries` somewhere else and pass that in directly here.  
It looks sort of weird to see `EntryQueries` in here :)

Reply:
@TimBeyer good point - done!